neuro_san/internals/interfaces/context_type_llm_factory.py

```diff
@@ -48,12 +50,12 @@ def load(self):
         """
         raise NotImplementedError
 
-    def create_llm(self, config: Dict[str, Any]) -> Any:
+    def create_llm(self, config: Dict[str, Any]) -> LangChainLlmResources:
         """
-        Create an llm instance BaseLanguageModel from the fully-specified llm config.
+        Create a llm instance BaseLanguageModel from the fully-specified llm config.
         :param config: The fully specified llm config from which the LLM instance
                     should be created.
-        :return: An llm instance native to the context type.
+        :return: An LangChainLlmResources instance containing a model native to the context type.
                 Can raise a ValueError if the config's class or model_name value is
```

Comment:
Change of interface - create_llm() now returns LangChainLlmResources = model + run-time resources.